### ISSUE TITLE
Gray image result after training UNet+Tensorflow backend

### ISSUE THREAD

**Issue by Maxfashko** [fchollet/keras#6498]
Hello! I use the "UNet" network for semantic segmentation. The network "UNet" was used in the competition "kaggle ultra nerve". An example of implementing this neural network architecture is here:
https://github.com/jocicmarko/ultrasound-nerve-segmentation

For training, I prepared the data from Toronto's dataset:
https://www.cs.toronto.edu/~vmnih/data/"

I pre-cut the image of the original size 1500 * 1500 into smaller pictures with the size 96 * 96 and converted them into the format "* npy".
As a result of learning the neural network on my data, after the 1st, 5th, 20th era, I run a test on the test images. I get a completely gray picture, instead of a binary mask.
What could be the reason for this prediction? If I use the data used by the author of the above code, I get binary masks. Does anyone have any idea what might be the reason?

pred:
![2_pred](https://cloud.githubusercontent.com/assets/11171626/25697810/a164f27a-30e6-11e7-9f56-eb28b06681cb.png)
img:
![1](https://cloud.githubusercontent.com/assets/11171626/25697899/f31c8b46-30e6-11e7-9e97-9769b22f0417.png)
label:
![1_mask](https://cloud.githubusercontent.com/assets/11171626/25697901/f82f4ab0-30e6-11e7-9d5b-54b3e39bd72f.png)

**Comment by Irtza** [comment]
@Maxfashko Did you figure it out ? I'm getting the same problem, while training the model on my DataSet

**Comment by AniketGurav** [comment]
Hi Maxfashko , I am facing similar problem.
I am trying to use encoder-decoder network for segmentation.
After validation loss converges I tried my model but I am also getting gray Image as a output.
Please help If you have solved this problem.

**Comment by Maxfashko** [comment]
@AniketGurav You need more learning epochs

**Comment by MainakBhattacharya** [comment]
@Maxfashko yup increasing the #epochs worked!
